GitUI/CommandsDialogs/SettingsDialog/Pages/FormBrowseRepoSettingsPage.Designer.cs

```diff
@@ -108,23 +115,59 @@ private void InitializeComponent()
             gbGeneral.Controls.Add(chkUseDiffViewerForBlame);
             gbGeneral.Location = new Point(11, 11);
             gbGeneral.Name = "gbGeneral";
-            gbGeneral.Size = new Size(435, 104);
+            gbGeneral.Size = new Size(1505, 104);
             gbGeneral.TabIndex = 1;
             gbGeneral.TabStop = false;
             gbGeneral.Text = "General";
             // 
             // gbTabs
             // 
             gbTabs.Anchor = AnchorStyles.Top | AnchorStyles.Left | AnchorStyles.Right;
+            gbTabs.Controls.Add(_NO_TRANSLATE_ProcessHistoryDepth);
+            gbTabs.Controls.Add(lblProcessHistoryDepth);
+            gbTabs.Controls.Add(chkProcessHistoryAsTab);
             gbTabs.Controls.Add(chkShowGpgInformation);
             gbTabs.Controls.Add(chkChowConsoleTab);
             gbTabs.Location = new Point(11, 121);
             gbTabs.Name = "gbTabs";
-            gbTabs.Size = new Size(435, 82);
+            gbTabs.Size = new Size(1505, 125);
             gbTabs.TabIndex = 5;
             gbTabs.TabStop = false;
             gbTabs.Text = "Tabs (restart required)";
             // 
+            // _NO_TRANSLATE_ProcessHistoryDepth
+            // 
+            _NO_TRANSLATE_ProcessHistoryDepth.Location = new Point(146, 92);
+            _NO_TRANSLATE_ProcessHistoryDepth.Maximum = new decimal(new int[] { 1000, 0, 0, 0 });
+            _NO_TRANSLATE_ProcessHistoryDepth.Name = "_NO_TRANSLATE_ProcessHistoryDepth";
+            _NO_TRANSLATE_ProcessHistoryDepth.Size = new Size(48, 23);
+            _NO_TRANSLATE_ProcessHistoryDepth.TabIndex = 10;
+            _NO_TRANSLATE_ProcessHistoryDepth.Value = new decimal(new int[] { 1000, 0, 0, 0 });
+            // 
+            // lblProcessHistoryDepth
+            // 
+            lblProcessHistoryDepth.AutoSize = true;
+            lblProcessHistoryDepth.Location = new Point(9, 94);
+            lblProcessHistoryDepth.Name = "lblProcessHistoryDepth";
+            lblProcessHistoryDepth.Size = new Size(123, 15);
+            lblProcessHistoryDepth.TabIndex = 9;
+            lblProcessHistoryDepth.Text = "Process history depth:";
+            // 
+            // chkProcessHistoryAsTab
+            // 
+            chkProcessHistoryAsTab.AutoSize = true;
+            chkProcessHistoryAsTab.AutoSizeMode = AutoSizeMode.GrowAndShrink;
+            chkProcessHistoryAsTab.Checked = false;
+            chkProcessHistoryAsTab.Location = new Point(9, 72);
+            chkProcessHistoryAsTab.Margin = new Padding(4, 3, 4, 3);
+            chkProcessHistoryAsTab.Name = "chkProcessHistoryAsTab";
+            chkProcessHistoryAsTab.Size = new Size(249, 19);
+            chkProcessHistoryAsTab.TabIndex = 8;
+            chkProcessHistoryAsTab.Text = "Process history as tab (otherwise as panel)";
```

Comment:
"Show Process history as tab (otherwise as panel)" or "Display Process history as tab (otherwise as panel)" (like all the settings in this setting panel)?